java/java/JavaParser.g4

```diff
@@ -582,9 +582,9 @@ expressionList
     ;
 
 methodCall
-    : identifier '(' expressionList? ')'
-    | THIS '(' expressionList? ')'
-    | SUPER '(' expressionList? ')'
+    : identifier arguments
+    | THIS arguments
+    | SUPER arguments
```

Comment:
Does it make sense to simplify it in the following way?

```suggestion
    : (identifier | THIS | SUPER) arguments
```

Reply:
Yes, it can make sense